sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -239,7 +239,7 @@ Licensed under the MIT License.
                 <include>com.fasterxml.jackson.core:jackson-annotations:[2.11.2]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-annotations;external_dependency} -->
                 <include>com.fasterxml.jackson.core:jackson-databind:[2.11.2]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
                 <include>com.fasterxml.jackson.datatype:jackson-datatype-jsr310:[2.11.2]</include> <!-- {x-include-update;com.fasterxml.jackson.datatype:jackson-datatype-jsr310;external_dependency} -->
-                <include>com.fasterxml.jackson.module:jackson-module-afterburner:[2.10.1]</include> <!-- {x-include-update;cosmos_com.fasterxml.jackson.module:jackson-module-afterburner;external_dependency} -->
+                <include>com.fasterxml.jackson.module:jackson-module-afterburner:[2.11.2]</include> <!-- {x-include-update;cosmos_com.fasterxml.jackson.module:jackson-module-afterburner;external_dependency} -->
```

Comment:
Looks like this was a copy and paste from `azure-cosmos` without this module actually using this dependency.

Reply:
Yes, we should get rid of it, good catch @alzimmermsft

Reply:
Can you please remove it in this PR ?

Reply:
Will remove it in the next commit